src/components/NavBar/NavBar.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { Layout, Menu } from 'antd';
+import { Link, useLocation } from 'react-router-dom';
+
+import {
+  ClockCircleFilled,
+  HomeFilled,
+  SafetyCertificateFilled,
+  DatabaseFilled,
+  BellFilled,
+} from '@ant-design/icons';
+
+import { ReactComponent as GSPLogo } from '../../assets/GSPLogo.svg';
```

Comment:
🚫 **[eslint]** <[import/no-unresolved](https://github.com/import-js/eslint-plugin-import/blob/v2.27.5/docs/rules/no-unresolved.md)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unable to resolve path to module '../../assets/GSPLogo.svg'.